code/src/java/plugin/exporttokens/DeityToken.java

```diff
@@ -152,7 +160,13 @@ else if ("FAVOREDWEAPON".equals(subTag))
 			}
 			else if ("PANTHEONLIST".equals(subTag))
 			{
-				retString = StringUtil.join(deity.getSafeListFor(ListKey.PANTHEON), ", ");
+				FactSetKey<String> fk = FactSetKey.valueOf("Pantheon");
```

Comment:
Basically, no. 

The impetus behind FactSet is that it is trying to remove code from the path of knowledge.  While we still have places that is violated, if we make it DATAFORMAT:PANTHEON in the Definition file, then we've reintroduced the concept of a Pantheon to the code.  Much better to have it a String (since the long term theory is the only place it's grabbed is in an OS anyway)